src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/layout.yaml

```diff
@@ -0,0 +1,8 @@
+twig_hooks:
+    hooks:
+        'sylius_admin.base#stylesheets':
+            styles:
+                template: '@SyliusAdmin/shared/layout/stylesheets.html.twig'
```

Comment:
I hesitate with '@SyliusAdmin/shared/layout/base/stylesheets.html.twig'

Reply:
@GSadee @Rafikooo Are you ok with that place?

Reply:
I would be for putting this template under the `base` directory 👌🏻